receiver/prometheusexecreceiver/go.mod

```diff
@@ -145,4 +145,6 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourceto
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/opencensus => ../../pkg/translator/opencensus
 
+replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheusremotewrite => ../../pkg/translator/prometheusremotewrite
```

Comment:
Is this necessary?  This module doesn't seem to be required.

Reply:
Same here:

```
╭─ ~/g/s/g/o/opentelemetry-collector-contrib/receiver/prometheusexecreceiver export-rw-converter !1
╰─❯ go mod why github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheusremotewrite
# github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheusremotewrite
github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusexecreceiver
github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver
github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver/internal
github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver/internal.test
github.com/open-telemetry/opentelemetry-collector-contrib/exporter/prometheusremotewriteexporter
github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheusremotewrite
```